bin/io/serializers.js

```diff
@@ -0,0 +1,29 @@
+import pino from 'pino';
+
+export function responseSerializer({ status, statusText, headers, url, _raw }) {
+  return {
+    status,
+    statusText,
+    headers,
+    url,
+    _raw: _raw.toString(),
+  };
+}
+
+export const errSerializer = err => {
```

Comment:
Note I am updating this in the monorepo right now

Reply:
what's my todo here?

Reply:
https://github.com/chromaui/chromatic/pull/2269/files

Reply:
✅